### PR TITLE
Display comments using Query

### PR DESCRIPTION
**Pull request by kristen-m**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -60,7 +60,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     commentEntity.setProperty("timestamp", timestamp);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     datastore.put(commentEntity);
-    //response.setContentType("text/html");
+    response.setContentType("text/html");
```
Comment (juliareichel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:61]: LGTM

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -32,47 +32,36 @@
...
 public class DataServlet extends HttpServlet {
-  private ArrayList<String> comments;
-
-
-  @Override
-  public void init(){
-    comments = new ArrayList<String>();    
-  }
-
+  private ArrayList<String> comments;
```
Comment (juliareichel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: no need to declare this as a member variable- can declare and initialize in the doGet function

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -32,47 +32,36 @@
...
+      long id = entity.getKey().getId();
+      String comment = (String) entity.getProperty("comment");
+      long timestamp = (long) entity.getProperty("timestamp");
+      comments.add(comment);
+    }
+    Gson gson = new Gson();
     response.setContentType("application/json;");
-    String json = new Gson().toJson(comments);
-    response.getWriter().println(json);
+    response.getWriter().println(gson.toJson(comments));
```
Comment (juliareichel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: i also liked this one liner (as opposed to separating the two)

**Review by Kaleab-Gebremichael — APPROVED** [review]
LGTM

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -24,6 +24,9 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
```
Comment (juliareichel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:29]: lgtm!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (kristen-m) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Thanks for the reminder! Totally forgot to change that

**Review by damargulis — APPROVED** [review]

**Pull request merged**